locales/spa.json

```diff
@@ -1,467 +1,453 @@
-      "invalid": "Código QR inválido",
-      "offline": "Por favor, conéctate a internet para escanear códigos QR usando el modo de intercambio en línea",
-      "sent": "La tarjeta ha sido enviada...",
-      "sentHint": "Esperando a que el receptor guarde o descarte tu tarjeta",
-      "sharing": {
-        "title": "Compartiendo...",
-        "hint": "Espere mientras compartimos la tarjeta elegida.",
-        "timeoutHint": "Está tomando más tiempo del esperado para compartir. Puede haber un problema con la conexión."
-      },
-      "accepted": {
-        "title": "¡Éxito!",
-        "message": "Tu tarjeta ha sido compartida exitosamente con el Verificador"
-      },
-      "rejected": {
-        "title": "Aviso",
-        "message": "Tu tarjeta ha sido descartada por el Verificador"
-      },
-      "bleError": {
-        "title": "Error al transferir",
-        "message": "Ocurrió un error al transferir la tarjeta. Por favor, inténtalo de nuevo.",
-        "hint": "Error: {{code}}"
-      }
-    }
-  },
-  "SelectVcOverlay": {
-    "header": "Compartir tarjeta",
-    "chooseVc": "Selecciona la tarjeta que deseas compartir",
-    "share": "Compartir",
-    "verifyAndShare": "Verificar identidad y compartir"
-  },
-  "SendVcScreen": {
-    "reasonForSharing": "Motivo de compartir (opcional)",
-    "acceptRequest": "Compartir",
-    "acceptRequestAndVerify": "Compartir con selfie",
-    "reject": "Rechazar",
-    "consentToPhotoVerification": "Doy mi consentimiento para que se tome mi foto para la autenticación"
-  },
-  "VerifyIdentityOverlay": {
-    "status": {
-      "verifyingIdentity": "Verificando identidad..."
-    },
-    "errors": {
-      "invalidIdentity": {
-        "title": "No se puede verificar la identidad",
-        "message": "No se reconoce el rostro. Por favor, inténtalo de nuevo.",
-        "messageNoRetry": "No se reconoce el rostro."
-      }
-    }
-  },
-  "WelcomeScreen": {
-    "title": "Solución de identidad de código abierto",
-    "getStarted": "Empezar",
-    "unlockApp": "Desbloquear aplicación",
-    "errors": {
-      "invalidateKeyError": {
-        "title": "Algunas pruebas de identidad se eliminan debido a problemas de seguridad.",
-        "message": "Vuelva a descargar."
-      }
-    }
-  },
-  "common": {
-    "cancel": "Cancelar",
-    "save": "Guardar",
-    "dismiss": "Cerrar",
-    "editLabel": "Editar {{label}}",
-    "tryAgain": "Intentar de nuevo",
-    "camera": {
-      "errors": {
-        "missingPermission": "Esta aplicación utiliza la cámara para escanear el código QR de otro dispositivo."
-      },
-      "allowAccess": "Permitir acceso a la cámara"
-    },
-    "errors": {
-      "genericError": "Algo salió mal. ¡Por favor, inténtalo de nuevo después de un tiempo!"
-    },
-    "biometricPopup": {
-      "title": "Desbloquear aplicación",
-      "description": "Utilice la huella digital para desbloquear la aplicación."
-    }
-  }
+	"ActivityLogText": {
```

Comment:
can we please bring the intendation back to exisiting alignment.

Reply:
Reverted and updated the files with new text in the Scan screen.